draft-ietf-quic-ack-frequency.md

```diff
@@ -289,9 +289,12 @@ acknowledgements can increase the time it takes for a sender to detect packet
 loss. The IMMEDIATE_ACK frame helps mitigate this problem.
 
 An IMMEDIATE_ACK frame can be useful in other situations as well. For example,
-it can be used with a PING frame (Section 19.2 of {{QUIC-TRANSPORT}}) if a
-sender wants an immediate RTT measurement or if a sender wants to establish
-receiver liveness as quickly as possible.
+it can be send in the same packet together with a PING frame
+(Section 19.2 of {{QUIC-TRANSPORT}}) if a sender wants an immediate RTT
+measurement or if a sender wants to establish receiver liveness as quickly
+as possible. PING frames are ack-eliciting but if a PING frame is sent without
+an IMMEDIATE_ACK frame, the receiver might not immediately send an ACK based
+on it local ACK strategy.
```

Comment:
```suggestion
it can be sent as an alternative of a PING frame
(Section 19.2 of {{QUIC-TRANSPORT}}) if a sender wants an immediate RTT
measurement or if a sender wants to establish receiver liveness as quickly
as possible. PING frames are ack-eliciting but
the receiver might not immediately send an ACK based
on it local ACK strategy.
```

Maybe state that IMMEDIATE_ACK frames can be used as an alternative? I think that was the point of @neild.